core/src/com/dragons/game/model/blockFactory/blocks/WallBlock.java

```diff
@@ -1,15 +1,25 @@
 package com.dragons.game.model.blockFactory.blocks;
 
+import com.badlogic.gdx.physics.box2d.World;
+import com.dragons.game.model.blockFactory.BlockType;
+import com.dragons.game.utilities.BodyBuilder;
 import com.dragons.game.view.modelViews.IModelObserver;
 
+import static com.dragons.game.utilities.ContactType.WALL;
+
 public class WallBlock extends Block{
 
-    public WallBlock(IModelObserver observer) {
+    // TODO: ADD TILE TO FIELD
+
+    public WallBlock(IModelObserver observer, World world, BlockType type) {
         super(observer);
+        body = BodyBuilder.createBoxBody(x, y, width, height, world, true, WALL, 0f);
```

Comment:
Jeg ser argumentet med å passe inn world for å lage bodies på denne måten, men det er et nokså grovt prinsippbrudd som bør unngås.